apollo-backend/src/test/java/io/logz/apollo/EnvironmentTest.java

```diff
@@ -67,4 +67,97 @@ public void testGetAllEnvironments() throws ApolloClientException {
+
+        ApolloTestClient apolloTestClient = Common.signupAndLogin();
+
+        // Add environments
+        Environment testEnvironment = createAndSubmitEnvironment(apolloTestClient, false);
+        Environment activeTestEnvironment = createAndSubmitEnvironment(apolloTestClient);
+
+        // Get all environments should return everything
+        List<Environment> allEnvironments = apolloTestClient.getAllEnvironments();
+
+        boolean foundNotActiveEnvironment = allEnvironments.stream()
+                .anyMatch(environment -> environment.getId().equals(testEnvironment.getId()));
+        boolean foundActiveEnvironment = allEnvironments.stream()
+                .anyMatch(environment -> environment.getId().equals(activeTestEnvironment.getId()));
+
+        assertThat(foundNotActiveEnvironment).isTrue();
+        assertThat(foundActiveEnvironment).isTrue();
+
+
+        List<Environment> allActiveEnvironments = apolloTestClient.getAllActiveEnvironments();
+
+        // Inactive environments should be excluded from search results
+        foundNotActiveEnvironment = allActiveEnvironments.stream()
+                .anyMatch(environment -> environment.getId().equals(testEnvironment.getId()));
+        assertThat(foundNotActiveEnvironment).isFalse();
+
+        // Active environment should be present in response
+        Optional<Environment> environmentFromApi = allActiveEnvironments.stream()
+                .filter(environment -> environment.getId().equals(activeTestEnvironment.getId()))
+                .findFirst();
+
+        assertThat(environmentFromApi).isPresent();
+
+        // Make sure we got the correct one
+        assertThat(environmentFromApi.get().getName()).isEqualTo(activeTestEnvironment.getName());
+        assertThat(environmentFromApi.get().getGeoRegion()).isEqualTo(activeTestEnvironment.getGeoRegion());
+        assertThat(environmentFromApi.get().getAvailability()).isEqualTo(activeTestEnvironment.getAvailability());
+        assertThat(environmentFromApi.get().getKubernetesMaster()).isEqualTo(activeTestEnvironment.getKubernetesMaster());
+        assertThat(environmentFromApi.get().getKubernetesToken()).contains("*");
+        assertThat(environmentFromApi.get().getKubernetesNamespace()).isEqualTo(activeTestEnvironment.getKubernetesNamespace());
+        assertThat(environmentFromApi.get().getIsActive()).isEqualTo(activeTestEnvironment.getIsActive());
+    }
+
+    @Test
+    public void testUpdateEnvironment() throws ApolloClientException {
+
+        ApolloTestClient apolloTestClient = Common.signupAndLogin();
+
+        // Add environment
+        Environment testEnvironment = createAndSubmitEnvironment(apolloTestClient);
+
+        // Check that environment is created
+        boolean foundTestEnvironment = apolloTestClient.getAllEnvironments().stream()
+                .anyMatch(environment -> environment.getId().equals(testEnvironment.getId()));
+        assertThat(foundTestEnvironment).isTrue();
```

Comment:
There is a different test for that, "testAddAndGetEnvironment", no need for this check here :)